### ISSUE TITLE
facet_wrap not working 

### ISSUE THREAD

**Issue by philly1993** [sportsdataverse/sportyR#26]
When trying to use `facet_wrap` only one facet appears with the rink

![image](https://github.com/sportsdataverse/sportyR/assets/112036822/a8aa3c41-67bc-4ebe-b665-624071f16bfa)

```
data <- tibble(x = c(0, 22, -22, 41),
               y = c(50, 25, 85, 41.3),
               group = c("shot", "shot", "goal", "goal"))

geom_hockey("NHL", "ozone", rotation = 90) +
  geom_point(data = data, aes(x = x, y = y, color = group)) +
  facet_wrap(~group)
```

**Comment by rossdrucker** [comment]
I've gotten the rink to appear on both facets by using `facet_grid(~group)` instead, however there seems to be an issue with the faceted data displaying correctly. I'm not sure what's going on with that quite yet, but will adjust as soon as I can figure it out. Everything as far as I can tell seems to match what I'd expect to happen when faceting this way

**Comment by rossdrucker** [comment]
@philly1993 Now addressed thanks to @mrcaseb on #27. Should be released in the next CRAN update, thanks for pointing this out!

**Comment by philly1993** [comment]
@rossdrucker Thanks! Just loaded up version of package via GitHub install and it works as I expected.  Thanks for the quick fix.
